### DISCUSSION TITLE
Is it possible to disassemble 3ds file formats in Ghidra?

### DISCUSSION THREAD

**Discussion by SirEnder125** [NationalSecurityAgency/ghidra#2696] [Q&A]
Hello. I was wondering if Ghidra could possible disassemble/decompile a 3ds executable (.3dsx) or a CTR Installable Archive (.cia).
Is it possible? Are any extensions available that can do this?

[[.cia file format]](https://www.3dbrew.org/wiki/CIA)

**Comment by ReversingWithMe** [comment]
https://github.com/Martmists-GH/ghidra-ctr-loader

Looks like the answer to this is yes now.
